node/silkworm/db/access_layer.cpp

```diff
@@ -502,22 +578,62 @@ std::optional<ChainConfig> read_chain_config(mdbx::txn& txn) {
     return ChainConfig::from_json(json);
 }
 
+static Bytes head_header_key() {
+    std::string table_name = db::table::kHeadHeader.name;
+    Bytes key{table_name.begin(), table_name.end()};
+    return key;
+}
+
+void write_head_header_hash(mdbx::txn& txn, const evmc::bytes32& hash) {
+    write_head_header_hash(txn, hash.bytes);
+}
+
 void write_head_header_hash(mdbx::txn& txn, const uint8_t (&hash)[kHashLength]) {
     Cursor target(txn, table::kHeadHeader);
-    mdbx::slice key(db::table::kLastHeaderKey);
-    target.upsert(key, to_slice(hash));
+    Bytes key = head_header_key();
+    auto skey = db::to_slice(key);
+
+    target.upsert(skey, to_slice(hash));
```

Comment:
The previous impl was simpler, less reduntant and saves 1 bytes and 1 byteview

Reply:
Same as previous